src/isomorphic/classic/element/ReactElementValidator.js

```diff
@@ -180,21 +180,52 @@ function validatePropTypes(element) {
   }
 }
 
+function warnInvalidType(inputValue) {
+  // warning(...) messages need to be string literals.
```

Comment:
Don’t worry about repeating the message, it doesn’t matter.
Please could you remove this comment?